docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -20,272 +20,136 @@ Current binder implementations include:
-
-*_batch-mode_*
-
-Whether to enable the entire batch of messages to be passed to the consumer function in a `List`.
-
-Default: `False`
-
-===== Event Hub Consumer Properties
-
-It supports the following configurations with the format of `spring.cloud.stream.eventhubs.bindings.&lt;channelName&gt;.consumer`.
-
-*_start-position_*
-
-Whether the consumer receives messages from the beginning or end of event hub. if `EARLIEST`, from beginning. If
-`LATEST`, from end.
-
-Default: `LATEST`
-
-*_checkpoint-mode_*
-
-The mode in which checkpoints are updated.
-
-`RECORD`, `default` mode. Checkpoints occur after each record is successfully processed by user-defined message
-handler without any exception. If you use `StorageAccount` as checkpoint store, this might become bottleneck.
-
-`BATCH`, checkpoints occur after each batch of messages successfully processed by user-defined message handler
-without any exception. Be aware that batch size could be any value and `BATCH` mode is only supported when consume
-batch
-mode is set true.
-
-`MANUAL`, checkpoints occur on demand by the user via the `Checkpointer`. You can do checkpoints after the message has been successfully processed. `Message.getHeaders.get(AzureHeaders.CHECKPOINTER)`callback can get you the `Checkpointer` you need. Please be aware all messages in the corresponding Event Hub partition before this message will be considered as successfully processed.
-
-`PARTITION_COUNT`, checkpoints occur after the count of messages defined by `checkpoint_count` successfully processed for each partition. You may experience reprocessing at most `checkpoint_count` of when message processing fails.
-
-`Time`, checkpoints occur at fixed time interval specified by `checkpoint_interval`. You may experience reprocessing of messages during this time interval when message processing fails.
-
-Default: `RECORD`
-
-NOTE: when consume batch mode is false(default value), `BATCH` checkpoint mode is not invalid.
-
-*_checkpoint-count_*
-
-Effectively only when `checkpoint-mode` is `PARTITION_COUNT`. Decides the amount of message for each partition to do one checkpoint.
-
-Default: `10`
-
-*_checkpoint-interval_*
-
-Effectively only when `checkpoint-mode` is `Time`. Decides The time interval to do one checkpoint.
-
-Default: `5s`
-
-*_max-size_*
-
-The maximum number of events that will be in the list of a message payload when the consumer callback is invoked.
-
-Default: `10`
-
-*_max-wait-time_*
-
-The max time `Duration` to wait to receive a batch of events up to the max batch size before invoking the consumer callback.
-
-Default: `null`
-
-To see the list of all Spring Cloud Azure related configuration properties please check link:appendix.html[the Appendix page].
-
 ==== Basic Usage
 
-==== Samples
-
-==== Error Channels
+. Method 1: Connection string based usage
 
-*_consumer error channel_*
+**  Create [Azure Event Hubs][create-event-hubs].
```

Comment:
This part of creating resources is unrelated with "Method 1", so you should move it to the upper part.